gems/pending/spec/util/mount/miq_generic_mount_session_spec.rb

```diff
@@ -2,25 +2,27 @@
 require 'util/mount/miq_generic_mount_session'
 
 describe MiqGenericMountSession do
-  context "#connect" do
-    before do
-      MiqGenericMountSession.stub(:raw_disconnect)
-      @s1 = MiqGenericMountSession.new({:uri => '/tmp/abc'})
-      @s2 = MiqGenericMountSession.new({:uri => '/tmp/abc'})
+  it "#connect returns a string pointing to the mount point" do
+    described_class.stub(:raw_disconnect)
+    s = described_class.new(:uri => '/tmp/abc')
+    s.logger = Logger.new("/dev/null")
 
-      @s1.logger = Logger.new("/dev/null")
-      @s2.logger = Logger.new("/dev/null")
+    expect(s.connect).to match(/\A\/tmp\/miq_\d{8}-\d{5}-\w+\z/)
```

Comment:
Prefer %r for these regexes that have embedded slashes...makes it a lot easier to read (even though it's just one.